### DISCUSSION TITLE
Off course navigation alert

### DISCUSSION THREAD

**Discussion by drazde** [osmandapp/OsmAnd#17922] [Ideas]
I found very usefull the app for planning and do my hiking activities.
When I walk I don't have the smartphone always on my hand and I don't want to watch it too much... 

So could happen then lose the track planned.
Is it possible to have a alert when rec are too different (100 meters or distance configurable) to the first track viewed?

For hiking and MTB activity this features could be very usefull
Thanks

![Screenshot_20230815_083224_OsmAnd1](https://github.com/osmandapp/OsmAnd/assets/2643448/e1f93d52-7478-467f-8c1d-9191df501d5a)

**Comment by DmitryAlexei** [comment]
Please use *Navigation by GPX* option (https://osmand.net/docs/user/navigation/setup/gpx-navigation) and configure voice prompts according to your needs (https://osmand.net/docs/user/navigation/setup/gpx-navigation).
For example, you can leave *announce deviation from the route* on and turn off other options in voice prompts.

**Comment by mikehgentry** [comment]
If you don't want navigation instructions, but only warnings for when you've gone off the route, the beep-minimal package might suit.

https://osmand.net/docs/user/navigation/guidance/voice-navigation#beep-modes

**Comment by drazde** [comment]
> Please use _Navigation by GPX_ option (https://osmand.net/docs/user/navigation/setup/gpx-navigation) and configure voice prompts according to your needs (https://osmand.net/docs/user/navigation/setup/gpx-navigation). For example, you can leave _announce deviation from the route_ on and turn off other options in voice prompts.

I honestly didn't know this features, thanks!
But the navigation could consume much more battery?

**Reply by mikehgentry** [reply]
It will to some extent, but you can mitigate it by keeping the screen off

https://osmand.net/docs/user/navigation/guidance/voice-navigation/#screen-control
